<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmbFileBrowser</class>
 <widget class="QDialog" name="SmbFileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QTableView" name="tableViewFM"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButtonBack">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>back</string>
       </property>
       <property name="icon">
        <iconset theme="go-previous">
         <normaloff>../../../../</normaloff>../../../../</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonUp">
       <property name="text">
        <string>Up</string>
       </property>
       <property name="icon">
        <iconset theme="go-up">
         <normaloff>../../../../</normaloff>../../../../</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPath">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
